cmd/osm-bootstrap/crds/config_mesh_root_certificate.yaml

```diff
@@ -48,19 +48,20 @@ spec:
             spec:
               type: object
               required:
+                - intent
                 - provider
               properties:
                 trustDomain:
                   description: Trust Domain to use in common name for certificates, e.g. "example.com"
                   type: string
                   default: cluster.local
                 intent:
-                  description: Intent specifies the role of the MeshRootCertificate, can be active or passive
+                  description: Intent specifies the role of the MeshRootCertificate, can be active, passive
                   type: string
                   enum:
                     - passive
                     - active
-                  default: passive
+                    - inactive
```

Comment:
I agree, `inactive` makes sense as the default. Can we add a default for the intent field even though in the last OSM release we included an MRC without the intent field? We aren't updating the resource version

Reply:
good question, I don't think we have logic to support conversion webhooks.  

reading https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting

It looks like 

```
Defaulting happens on the object

in the request to the API server using the request version defaults,
when reading from etcd using the storage version defaults,
after mutating admission plugins with non-empty patches using the admission webhook object version defaults.
``` 

Which means if someone was using it they might end up with inactive certificate I think but something like this might happen anyways, since we wouldn't be able to read the value properly.

Reply:
Can `default` and `required` be used interchangeably here since we specify an enum and therefore limit the optional values?

Reply:
yea maybe don't need default since it is required field.  Works for me to leave it as is

Reply:
Will think about it more, but leaving it as is for now.